tests/flow/test_replication.py

```
@@ -167,6 +167,10 @@ def test_CRUD_replication(self):
 
         # the WAIT command forces master slave sync to complete
         source_con.execute_command("WAIT", "1", "0")
+        
+        # TODO: check if this sync is needed
+        wait_for_indices_to_sync(src)
```

Ruff (0.12.2)

`wait_for_indices_to_sync` may be undefined, or defined from star imports

(F405)